File: HaykAmirjanyan/pset6/cash/cash.py
from cs50 import get_float
from math import floor
money = -1.0
while money < 0:
    money = get_float("Give me my money! ")
money = round(money * 100)
count = floor(money / 25) + floor((money % 25) / 10) + floor(((money % 25) %
                                                              10) / 5) + floor((((money % 25) % 10) % 5) / 3) + (((money % 25) % 10) % 5) % 3
print(count)

File: Harut/workspace/week1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main()
{
    float d;

    do
    {
        d = get_float("dollar: ");
    }

    while (d < 0);

    int c = round(d * 100);
    int q = c / 25 + c % 25 / 10 + c % 25 % 10 / 5 + c % 25 % 10 % 5 / 1;
    printf("%i\n", q);
}